<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>421</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>保存文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openFileBtn">
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openFilesBtn">
     <property name="text">
      <string>打开多个文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openDirBtn">
     <property name="text">
      <string>打开文件夹</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
